amp-camp/src/server/server.js

```diff
@@ -36,6 +43,19 @@ const listener = app.listen(port, () => {
     console.log('App listening on port ' + listener.address().port);
 });
 
+//Intercepts urls for static pages, and calls 'renderPage' to inject the canonical tag
```

Comment:
Can you rewrite this comment to be clearer? I don't think you need to say that we call 'renderPage', since that's in the code - or that you have to say we "intercept" URLs, since that's what happens in express anyway. More what we're doing and why we're doing it in a couple of sentences!

Reply:
In general, comments are made with different formats and not always clear. 
I used some simpler ones for this task, but I was planning to re-write all of them when the refactor takes place (which is the next task).
I can make changes on this anyways.